src/dc_receive_imf.rs

```diff
@@ -541,6 +545,19 @@ async fn add_parts(
             }
         }
 
+        // if contact renaming is prevented (for mailinglists and bots),
+        // we use name from From:-header as override name
+        if prevent_rename {
+            if let Some(from) = mime_parser.from.first() {
+                if let Some(name) = &from.display_name {
+                    let name = name.trim_start_matches('~').trim_start();
```

Comment:
Why `trim_start_matches` for `~` is still here?

Reply:
opps :)